library/Core/autoload.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
+use function Baka\appPath;
 use Dotenv\Dotenv;
 use Phalcon\Loader;
-use function Canvas\Core\appPath;
 
 // Register the auto loader
-require dirname(dirname(__DIR__)) . DIRECTORY_SEPARATOR . '/vendor/canvas/core/src/Core/functions.php';
+require dirname(dirname(__DIR__)) . DIRECTORY_SEPARATOR . 'vendor/baka/baka/src/functions.php';
```

Comment:
@rwhite27 lets make the changes to the autoload so we dont need the function requirement